modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/java/JavaDeclarationDelegateTestCase.java

```diff
@@ -373,7 +373,7 @@ public void flowListeningOperationWithoutAttributes() {
 
     OperationDeclaration operation = getOperation(getConfiguration(declaration, BANANA), "getLunch");
     assertThat(operation, is(notNullValue()));
-    assertOutputType(operation.getOutput(), toMetadataType(Fruit.class), false);
```

Comment:
Is this because the parser now correctly sets the dynamic boolean, and before it had to wait for the enricher to do it?